src/components/shared/__tests__/AddRemoveForm.spec.js

```diff
@@ -35,31 +36,30 @@ describe('<AddRemoveForm />', () => {
     expect(results).toBe(mockData.terms.items)
   })
 
-  it('should use props.labelProp on button', (done) => {
+  it('should use props.labelProp on button', async () => {
     // confirms labelProp default of 'name' and custom value 'title'
     let results = wrapper.find({ 'data-test': 'item-button' })
     expect(results.first().render().text()).toBe(mockData.terms.items[0].name)
     
-    wrapper = shallow(<AddRemoveForm {...mockProps.questions} />)
-    setTimeout(() => {
-      results = wrapper.find({ 'data-test': 'item-button' })
-      expect(results.first().render().text()).toBe(mockData.questions.items[0].title)
-      done()
-    })
+    wrapper = await shallow(<AddRemoveForm {...mockProps.questions} />)
+    await wrapper.update()
+
+    results = wrapper.find({ 'data-test': 'item-button' })
+    expect(results.first().render().text()).toBe(mockData.questions.items[0].title)
```

Comment:
Our cleaner implementation of testing async code. Notice we first added `async` to this whole callback in `it()`. Now we can use `await` inside our code, which allows us to wait for async code to finish, then enabling us to run `expect` as normal. Writing it this way solves two main problems:

1. We don't have to write a `setTimeout()` and callback to wait for async to finish
1. We now have access to normal debugging during failures, which was hidden behind a timeout error with `setTimeout()`